python/ray/services.py

```diff
@@ -54,6 +54,8 @@
     "core/src/ray/raylet/raylet_monitor")
 RAYLET_EXECUTABLE = os.path.join(
     os.path.abspath(os.path.dirname(__file__)), "core/src/ray/raylet/raylet")
+GCS_SERVER_EXECUTABLE = os.path.join(
```

Comment:
We should copy the `gcsserver` binary to wheel in `setup.py`

Reply:
We have copy gcsserver in setup.py now.